### PR TITLE
Editor support for UPLC files

### PR DESCRIPTION
**Pull request by ChristianoBraga**
- Create `editor-support/emacs`.
- Move `uplc-mode.el` there.
- Add `README.md`

### CODE REVIEW

**Review by gtrepta — COMMENTED** [review]
You will probably want to rebase this off of #337 and make the changes you want.

**Comment by ChristianoBraga** [comment]
> You will probably want to rebase this off of #337 and make the changes you want.

Why can't we have a new PR for this?

**Comment by gtrepta** [comment]
> Why can't we have a new PR for this?

What do you mean? You already merged a PR with some emacs support, and this PR wasn't based off of master after that merge when I commented.

You have two copies of `uplc-mode.el` on this branch now, one at the base directory and one under `editor-support/`. I'll be happy to approve this PR if you remove the copy in the base directory.

**Comment by ChristianoBraga** [comment]
> I'll be happy to approve this PR if you remove the copy in the base
directory.

Ok now I understand.

Em qua., 12 de out. de 2022 às 12:48, gtrepta ***@***.***>
escreveu:

File: editor-support/emacs/README.md
Comment (gtrepta) [inline editor-support/emacs/README.md:20]:
```suggestion
4. Position the cursor near a `[` or `(`. Then press `C-c RET`. That
   "block" will be replaced by `...`. UPLC major mode uses `yafolding`
   which collapses thunks of text based on _indentation_. For example,
```

**Review by gtrepta — APPROVED** [review]

**Pull request merged**
